assets/src/blocks/PostsList/PostsListBlock.js

```diff
@@ -0,0 +1,70 @@
+export const POSTS_LIST_BLOCK = 'planet4-blocks/posts-list-block';
+
+export const registerPostsListBlock = () => {
+  const {registerBlockVariation} = wp.blocks;
+  const {__} = wp.i18n;
+
+  return registerBlockVariation('core/query', {
```

Comment:
If you add `category: 'planet4-blocks-beta'` then it will show in the editor in the blocks list along with our other beta blocks, which I believe is what we want?

<img width="267" alt="Screenshot 2023-11-16 at 11 00 26" src="https://github.com/greenpeace/planet4-master-theme/assets/6949075/6a343802-2d6d-4ade-99e4-0f8243182460">

Reply:
(not sure if we should add `(beta`) to the name 🤔)